### ISSUE TITLE
pyomo 5.6.2 import error

### ISSUE THREAD

**Issue by jankaeh** [PyPSA/PyPSA#69]
It seems with version 5.6.2 there has been another change at pyomos LConstraint object.
I just installed PyPSA 13.2 with the newest pyomo and got an import error because the workaround import for pyomo5.6.1 not exist anymore:
from pyomo.core.expr import expr_pyomo5

A downgrade to pyomo==5.6.1 helps, if anyone runs into this issue.

**Comment by coroa** [comment]
Same as #64. Has been fixed by: https://github.com/PyPSA/PyPSA/commit/8d9ce44518d7c43757c4528cd49675ccaff02d84

@nworbmot There should probably be a new release! The `cartopy` changes did also settle.

@FabianHofmann They did, didn't they?

**Comment by coroa** [comment]
To install `pypsa` from the `master` branch use:

```
pip install --upgrade git+https://github.com/PyPSA/pypsa.git#egg=pypsa
```

I'll leave this open, so others can find this issue quickly, until there is a new release.

**Comment by fneum** [comment]
@coroa `cartopy` works well for me with different projections and pie charts did not turn oval!

**Comment by FabianHofmann** [comment]
@coroa @fneum I can confirm. We just doublechecked on another machine, works as expected.

**Comment by coroa** [comment]
Fixed in [release 0.14.0](https://github.com/PyPSA/PyPSA/releases/tag/v0.14.0)
